rules/windows/defense_evasion_dns_over_https_enabled.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/07/22"
+maturity = "production"
+updated_date = "2021/08/27"
+
+[rule]
+author = ["Austin Songer"]
+description = "Identifies when a user enables DNS-over-HTTPS. This can be used to hide internet activity or be used to hide
+the process of exfiltrating data. With this enabled organization will lose visibility into data such as query type, response
+and originating IP that are used to determine bad actors."
+
+from = "now-30m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "DNS-over-HTTPS Enabled by Registry"
```

Comment:
```suggestion
name = "DNS-over-HTTPS Enabled via Registry"
```

Reply:
This one still needs to be applied, unresolving